### DISCUSSION TITLE
How can I determine rcut_smth for my system?

### DISCUSSION THREAD

**Discussion by saeedshahi** [deepmodeling/deepmd-kit#1675] [Q&A] [answered]
Hi,
Is there any relationship between rcut_smth and the minimum distance between atoms in the system? Is there a systematic method to determine rcut_smth for a specific system? like in the recent DeepMD paper about DPLR water, they use rcut_smth = 4 and rcut = 6, instead of rcut_smth =0.5 which is the default one.

Best,
Saeed

**Comment by njzjz** [comment, marked as answer]
There is little difference in results between `rcut_smth = 4` and `rcut_smth = 0.5`.
